### DISCUSSION TITLE
we connect graphql in next js using pimcore  graphql getting a CORS error

### DISCUSSION THREAD

**Discussion by sawarmaljat** [pimcore/pimcore#16993] [Data-Hub]
Hello everyone!
we connect graphql in next js using pimcore  graphql getting a CORS error. how fix this issusse
IAccess to fetch at 'https://mcg-dw.com/pimcore-graphql-webservices/home' from origin 'https://dev-dw.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Comment by SuryaWebfox** [comment]
We're having similar issues when connecting from localhost (using React) to our staging GraphQL API

**Comment by dpfaffenbauer** [comment]
install nelmio/cors-bundle and configure the api path for CORS
